isso/js/app/i18n.js

```diff
@@ -90,7 +100,8 @@ define(["app/config", "app/i18n/bg", "app/i18n/cs", "app/i18n/da",
         vi: vi,
         zh: zh_CN,
         zh_CN: zh_CN,
-        zh_TW: zh_TW
+        zh_TW: zh_TW,
+        sk: sk
```

Comment:
Please keep this dictionary alphabetical.

Reply:
Reordered